tests/test_app.py

```diff
@@ -0,0 +1,49 @@
+import asyncio
+import time
+
+import pytest
+
+from textual.app import App
+
+
+@pytest.mark.asyncio
+async def test_execute_in_thread():
```

Comment:
I didn't see any tests applied to App or MessagePump directly, so I wasn't sure what the best way to write them is, this is just what I came up with but I'm open to better options

Reply:
Changed this to test `MessagePump` directly now since it's a public class anyways